dockers/docker-fpm-frr/base_image_files/rvtysh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# The command rvtysh can be run as root priviledge by any user without password, only allow to execute readonly commands.
+
+# The options in the show command cannot contains any charactors to run multiple sub-commands potentially, such as "\n", "\r", "|", "&", "$" and ";".
+if printf -- "$*" | grep -qPz '[\n\r|&$;]'; then
+    echo "Not allow to run the command, please use the comand 'sudo vtysh' instead." 1>&2
+    exit 1
+fi
+
+# The sub commands must start with "show"
+LAST_PARA=
+for para in "$@"
+do
+    if [ "$LAST_PARA" == "-c" ] && [[ "$para" != show*  ]]; then
+        echo "Not allow to run the command '$para', please use the comand 'sudo vtysh' instead." 1>&2
+        exit 1
+    fi
+    LAST_PARA=$para
```

Comment:
Suggest replacing `PARA/para` with either `PARAM/param` or `ARG/arg`

Reply:
Since this PR is adjusting a master branch PR for 201811. Could you fix on master first?

Reply:
Fixed, thanks, @jleveque .
@qiluo-msft , I have sent a new PR to fix in master. And updated on the PR for 201911.
PRs to Review:
Master/202012: https://github.com/Azure/sonic-buildimage/pull/7648
201911: https://github.com/Azure/sonic-buildimage/pull/7572
Please help review again, thanks.